protocol/synthetix/subgraph/package.json

```
@@ -16,6 +16,8 @@
     "deploy:optimism-goerli": "graph deploy snx-v3/optimism-goerli subgraph.optimism-goerli.yaml --output-dir ./build/optimism-goerli --product hosted-service",
     "alchemy:base-mainnet-andromeda": "graph deploy synthetix-base-mainnet-andromeda subgraph.base-mainnet-andromeda.yaml --output-dir ./build/base-mainnet-andromeda --version-label v1 --node https://subgraphs.alchemy.com/api/subgraphs/deploy --deploy-key $SATSUMA_KEY --ipfs https://ipfs.satsuma.xyz",
     "alchemy:base-sepolia-andromeda": "graph deploy synthetix-base-sepolia-andromeda subgraph.base-sepolia-andromeda.yaml --output-dir ./build/base-sepolia-andromeda --version-label v1 --node https://subgraphs.alchemy.com/api/subgraphs/deploy --deploy-key $SATSUMA_KEY --ipfs https://ipfs.satsuma.xyz",
+    "alchemy:arbitrum-one": "graph deploy synthetix-arbitrum-one subgraph.arbitrum-one.yaml --output-dir ./build/arbitrum-one --version-label v1 --node https://subgraphs.alchemy.com/api/subgraphs/deploy --deploy-key $SATSUMA_KEY --ipfs https://ipfs.satsuma.xyz",
```

Lets keep the naming consistent. arbitrum-one -> arbitrum-mainnet please.
```suggestion
    "alchemy:arbitrum-mainnet": "graph deploy synthetix-arbitrum-mainnet subgraph.arbitrum-mainnet.yaml --output-dir ./build/arbitrum-mainnet --version-label v1 --node https://subgraphs.alchemy.com/api/subgraphs/deploy --deploy-key $SATSUMA_KEY --ipfs https://ipfs.satsuma.xyz",
```

---

Ok yeah I'm fine with that

---

Updated